src/color/mfcamerareader.cpp

```diff
@@ -953,56 +946,32 @@ int CMFCameraReader::GetStride()
 
 k4a_result_t CMFCameraReader::CreateImage(CFrameContext *pFrameContext, k4a_image_t *image)
 {
-    void *context;
-    mf_buffer_wrapper_t *wrapper;
-
-    // We use a wrapper here so that we can keep track of the MF frame buffers in use; ALLOCATION_SOURCE_COLOR will
-    // count outstanding allocations. If too many are used then MF stops receiving images over USB until the captures
-    // are released.
-    wrapper = (mf_buffer_wrapper_t *)allocator_alloc(ALLOCATION_SOURCE_COLOR, sizeof(mf_buffer_wrapper_t), &context);
-
-    wrapper->allocator_context = context;
-    wrapper->pFrameContext = pFrameContext;
-
     return TRACE_CALL(image_create_from_buffer(m_image_format,
                                                m_width_pixels,
                                                m_height_pixels,
                                                GetStride(),
                                                pFrameContext->GetBuffer(),
                                                pFrameContext->GetFrameSize(),
                                                FrameDestroyCallback,
-                                               wrapper,
+                                               pFrameContext,
                                                image));
 }
```

Comment:
I think I see what you're trying to achieve here. In this case there is no "allocation" as far as the allocator is concerned. Adding an extra level of context redirects for the purpose of adding an object visible to the debugger is something I'd like to avoid. Maybe we just need an allocator_count_object(ALLOCATOR_SOURCE_MF_BUFFER) or similar just to increment/decrement the counter?